Soomla/Assets/Plugins/Soomla/Core/Config/CoreSettings.cs

```diff
@@ -42,6 +43,9 @@ public class CoreSettings : ISoomlaSettings
 		static CoreSettings()
 		{
 			SoomlaEditorScript.addSettings(instance);
+
+			List<string> additionalDependFiles = new List<string>(); //Add files that not tracked in file_list
```

Comment:
@Blinkq, please remove this instantiation and put an empty array directly in method invocation. SOOMLA will split instantiation, if additional files will be added.
for example,
`additionalDependFiles.ToArray() => new string[] {}`